docs/book/view-helpers.md

```diff
@@ -7,6 +7,29 @@ displaying translated content.
 See the [zend-view helpers documentation](http://framework.zend.com/manual/current/en/modules/zend.view.helpers.html#zend-view-helpers)
 for more information.
 
+## IDE auto-completion in templates
```

Comment:
I think, we should set the entire section as a blockquote.

```
> ### IDE auto-completion in templates
>
> The `Zend\I18n\View\Helper\HelperTrait` trait can be used to provide...
```

The headline is then marked up with `###`.